packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -216,21 +221,20 @@ function inputCellFactory(
             const minCell = parseCellKey(initialSelection[0]);
             const maxCell = parseCellKey(initialSelection[initialSelection.length - 1]);
             borderMask = computeBorderMask(
-                minCell.colIdx,
-                maxCell.colIdx,
+                orderedColumns.indexOf(minCell.fieldKey),
+                orderedColumns.indexOf(maxCell.fieldKey),
                 minCell.rowIdx,
                 maxCell.rowIdx,
                 colIdx,
-                rn,
+                rowIdx,
                 borderMask
             );
-            inSelection = initialSelection.includes(genCellKey(colIdx, rn));
+            inSelection = initialSelection.includes(genCellKey(fieldKey, rowIdx));
         }
 
-        const focused = editorModel.isFocused(colIdx, rn);
-        const className = classNames({ 'grid-col-with-width': hasCellWidthOverride(columnMetadata) });
-        const style = { textAlign: columnMetadata?.align ?? c.align ?? 'left' } as any;
-
+        // TODO: It would be really nice to drop the "row" prop in cell below, however the AssayTaskInputRenderer needs
```

Comment:
I'm not so sure about this. There are cases where an individual cell may need to be able to see other values from the row. What would be the recourse if the cell didn't receive the data row?

Reply:
The only case we have that needs the whole row is AssayTaskInputRenderer, it feels odd to have a prop that gets threaded through all of our Cell components, but is only needed by a single input renderer.